revista-ladinia.csl

```diff
@@ -0,0 +1,154 @@
+<?xml version="1.0" encoding="utf-8"?>
+<style xmlns="http://purl.org/net/xbiblio/csl" class="in-text" version="1.0">
+  <info>
+    <title>Revista Ladinia</title>
+    <title-short>LADINIA</title-short>
+    <id>http://www.zotero.org/styles/revista-ladinia</id>
+    <link href="http://www.zotero.org/styles/revista-ladinia" rel="self"/>
+    <link href="http://www.zotero.org/styles/apa" rel="template"/>
+    <link href="https://micura.it/la/ativites/ladinia/style-sheet" rel="documentation"/>
+    <author>
+      <name>Samuel Frontull</name>
+      <email>samuel.frontull@uibk.ac.at</email>
+    </author>
+    <author>
+      <name>Paolo Anvidalfarei</name>
+      <email>paolo@micura.it</email>
+    </author>
+    <category citation-format="author-date"/>
+    <category field="linguistics"/>
+    <issn>1124-1004</issn>
+    <updated>2022-04-25T14:01:00+01:00</updated>
+    <rights license="http://creativecommons.org/licenses/by-sa/3.0/">This work is licensed under a Creative Commons Attribution-ShareAlike 3.0 License</rights>
+  </info>
+  <locale xml:lang="it">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="anonymous">N.N.</term>
+      <term name="volume">vol.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <locale xml:lang="de">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="anonymous">N.N.</term>
+      <term name="volume">vol.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <locale xml:lang="en">
+    <terms>
+      <term name="and others"/>
+      <term name="no date">s.a.</term>
+      <term name="no-place">s.l.</term>
+      <term name="anonymous">N.N.</term>
+      <term name="volume">vol.</term>
+      <term name="editor">
+        <single>ed.</single>
+        <multiple>eds.</multiple>
+      </term>
+    </terms>
+  </locale>
+  <macro name="author">
+    <names variable="author">
+      <name name-as-sort-order="all" delimiter="/" form="short">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <et-al term="and others"/>
+      <substitute>
+        <names variable="editor"/>
+        <text term="anonymous"/>
+      </substitute>
+    </names>
+  </macro>
+  <macro name="author-bibliography">
+    <names variable="author">
+      <name name-as-sort-order="all" delimiter="/">
+        <name-part name="family" font-variant="small-caps"/>
+      </name>
+      <et-al/>
+      <substitute>
+        <names variable="editor translator">
```

Comment:
you can handle this simpler by adding the label to author above (where it doesn't appear bc author doesn't have a label) and then just use `<names variable="editor translator"/>` in substitute

Reply:
much more elegant, thanks!